### DISCUSSION TITLE
Expo 52 babel.config.js and metro.config.js

### DISCUSSION THREAD

**Discussion by terrico87** [software-mansion/react-native-reanimated#6726] [Q&A]
Hello,
quick one, is it still required to create the babel.config.js and metro.config.js files in expo 52? Not sure both Reanimated and Expo documentations are aligned.

Since after migrating to 52 I'm experience a big loss of smoothness in animations, so I thought that must be because I removed the babel file...

By the way now that I'm checking my old app in Expo 51, the one with the babel file, I've noticed that it doesn't explicitely mention reanimated but it goes as follow:

module.exports = function (api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
  };
};

which I believe is exactly what the expo documentation is stating:
"No additional configuration is required. Reanimated Babel plugin is automatically configured in babel-preset-expo when you install the library."

So, to sum up, questions:
1. Should I add the two files? Both or Babel is enough?
2. If so, for babel, should I explicitely add "plugins: ['react-native-reanimated/plugin']" or "presets: ['babel-preset-expo']" is enough?

Thanks
